examples/express/src/pages/App.jsx

```diff
@@ -0,0 +1,15 @@
+// pages/index.js
```

Comment:
To remove or to sync with the actual file name.

Reply:
I'll remove it